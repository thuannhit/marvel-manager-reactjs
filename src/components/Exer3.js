import React, {Component} from 'react';
import Header from './marvelComponents/header/Header';
import Table from './marvelComponents/table/Table';

class Exer3 extends Component {
  
    render() {
      return (
        <div className="App">
          <Header />
          <Table />
        </div>
      );
    }
  }
  
  export default Exer3;