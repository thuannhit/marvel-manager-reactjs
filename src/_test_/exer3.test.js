import React from 'react';
import Exer3 from '../components/Exer3';
import renderer from 'react-test-renderer';

jest.mock('../components/marvelComponents/table/Table', () => () => <div id="TableComponent">TableComponent</div>);
jest.mock('../components/marvelComponents/header/Header', () => () => <div id="HeaderComponent">HeaderComponent</div>);

it('Exer3 renders correctly', () => {
    const tree = renderer
        .create(<Exer3/>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
