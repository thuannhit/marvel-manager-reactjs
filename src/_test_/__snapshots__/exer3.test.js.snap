// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Exer3 renders correctly 1`] = `
<div
  className="App"
>
  <div
    id="HeaderComponent"
  >
    HeaderComponent
  </div>
  <div
    id="TableComponent"
  >
    TableComponent
  </div>
</div>
`;

exports[`renders correctly 1`] = `
<div
  className="App"
>
  <div
    id="HeaderComponent"
  >
    HeaderComponent
  </div>
  <div
    id="TableComponent"
  >
    TableComponent
  </div>
</div>
`;
